doc/updating_documentation/How_to_update_this_documentation.md

```diff
@@ -0,0 +1,26 @@
+# How to update this documentation
+
+## Documentation update procedure
+
+1. Make a pull request to master branch in repository Juniper/contrail-windows-docs with required changes.
+2. Once PR is accepted, an Administrator should deploy new modifications to gh-pages branch.
```

Comment:
According to this tutorial on technical writing: https://en.wikiversity.org/wiki/Technical_writing_overview_EE#Technical_writing_rules

one should use 'Active voice'

> Use active voice.

This means, no 'shoulds' or 'woulds'. Instead, we write what is the action that needs to be performed. Example: 'Contact the administrator blabla'